test/case/infix_dhcp/dhcp_routes/test.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python3
-"""
-DHCP option 121 vs option 3
+"""DHCP option 121 vs option 3
 
-Verify DHCP option 121 (static routes) is used over option 3 and that the
-routes exist in the operational datastore.
+Verify that DHCP option 121 (classless static routes) is used over the
+older option 3 (default gateway) in the DHCP client, see RFC3442.  Use
```

Comment:
Perhaps change
_in the DHCP client, see RFC3442._
to
_in the DHCP client when both are offered, see RFC3442._

Reply:
thx